### PR TITLE
Feature : lesson01

### PR DESCRIPTION
**Pull request by ja0112hk**
# [lesson1](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#1%E3%81%93%E3%81%AEdom%E3%82%92html%E5%86%85%E3%81%AEul%E3%81%AE%E4%B8%AD%E3%81%AB%E5%B7%AE%E3%81%97%E8%BE%BC%E3%82%93%E3%81%A7%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84)
## concerns and areas to focus on
- I chose appendChild.I was worried about insertBefore, but which one is fine?

Thank you for the review.

### CODE REVIEW

**Comment by ja0112hk** [comment]
## latest commit: a52185e

I have one question.
- Can I edit the HTML?
- If I can't edit, do I have to use "querySelector" in "script .js"?

I have corrected it, so please for the review again.

File: taks1/scrirpt.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const liElement = document.createElement('li');
+
+const liContent = document.createTextNode('これです');
```
Comment (kenmori) [inline taks1/scrirpt.js]:
```suggestion
const textNode = document.createTextNode('これです');
```

File: taks1/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <ul id="ul"></ul>
```
Comment (kenmori) [inline taks1/index.html]:
```suggestion
      <ul id="js-ul"></ul>
```

File: taks1/style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+li {
+    color: red;
+}
```
Comment (kenmori) [inline taks1/style.css]: remove style. no need.

**Comment by kenmori** [comment]
> Can I edit the HTML?

yes

**Comment by ja0112hk** [comment]
## latest commit: 447517e

I corrected.
Thank you for the review.

**Comment by kenmori** [comment]
approve

**Pull request merged**